applications/vanilla/settings/structure.php

```diff
@@ -423,6 +428,15 @@
         ->put();
 }
 
+if (!$captureOnly) {
```

Comment:
Maybe we could put the exist check in here since they are only used if captureOnly is false?